Makefile.am

```diff
@@ -347,8 +347,11 @@ COMMITTED_DIST = \
 # Build up the distribution using $COMMITTED_DIST and include node_modules and bower licenses
 # also automatically update minimum base dependency in RPM spec file
 dist-hook::
-	( cd $(srcdir); git ls-tree HEAD --name-only -r $(COMMITTED_DIST) || (echo $(COMMITTED_DIST) | tr ' ' '\n' ) ) | \
-		tar -C $(srcdir) -cf - -T - | tar -C $(distdir) -xf -
+	( if [ -d $(srcdir)/.git ]; then \
+		git -C $(srcdir) ls-tree HEAD --name-only -r $(COMMITTED_DIST) | tar -C $(srcdir) -cf - -T -; \
+	else \
+		tar -C $(srcdir) -cf - $(COMMITTED_DIST); \
```

Comment:
I like that last idea ... will do. But as far as I understand it's the outer ```make dist``` tarball that's distributed, not the second inner ```make distcheck``` test one. So this isn't really a big issue either way. But nice for correctness.